apim-apk-agent/internal/synchronizer/apis_fetcher.go

```diff
@@ -151,6 +153,47 @@ func FetchAPIsOnEvent(conf *config.Config, apiUUIDList []string, k8sClient clien
 				// Todo: Read the apis.zip and extract the api.zip,deployments.json
 			}
 
+			if err != nil {
+				logger.LoggerMsg.Error("Error while reading zip", err)
+			}
+
+			artifact, decodingError := transformer.DecodeAPIArtifact(data.Resp)
+
+			if decodingError != nil {
+				logger.LoggerSync.Error("Error while decoding the API Project Artifact:", decodingError)
+				return
+			}
+
+			apkConf, _, apkErr := transformer.GenerateAPKConf(artifact.APIJson)
+			logger.LoggerSync.Infof(apkConf)
+
+			if apkErr != nil {
+				logger.LoggerSync.Error("Error while generating apk-conf:", apkErr)
+				return
+			}
+
+			k8ResourceEndpoint := conf.DataPlane.K8ResourceEndpoint
+
+			deploymentDescriptor, err := transformer.ProcessDeploymentDescriptor([]byte(artifact.DeploymentDescriptor))
+			if err != nil {
+				logger.LoggerSync.Error("Error while decoding the API Project Artifact:", err)
+				return
+			}
+
+			crResponse, err := transformer.GenerateUpdatedCRs(apkConf, artifact.Swagger, k8ResourceEndpoint, deploymentDescriptor)
+
+			if err != nil {
+				logger.LoggerSync.Error("Error occured in receiving the updated CRDs:", err)
+				return
+			}
+
+			err = transformer.SaveZipToFile(crResponse, "/home/wso2/tmp/CRSet.zip")
```

Comment:
Why this path is hard coded? Should not we get the path from config? 
Even if it not something that comes from config we should define a variable for this path string and use it here.